File: exercice01/main.java
class Playground {
    public static void main(String[ ] args) {
        Book toup = new Book("Toup", 11);
        System.out.println(toup.title + " " + toup.price);
    }
}

class Book {

  public  String title;
  public int price;

  public Book(String title, int price) {
    this.title = title;
    this.price = price;
  }
}

File: exercice01/main.php
<?php
class Book
{

    public $title;
    public $price;

    function __construct($title, $price)
    {
        $this->title = $title;
        $this->price = $price;
    }
}

$toup = new Book("Toup", 11);

echo ($toup->title . " " . $toup->price);
//print_r($toup);
